src/jsonpath.d.ts

```diff
@@ -54,7 +58,7 @@ declare module 'jsonpath-plus' {
      * (Note that the current path and value will also be available to those
      *   expressions; see the Syntax section for details.)
      */
-    sandbox?: Map<string, any>
+    sandbox?: { [key: string]: any }
```

Comment:
While valid, there's also the following types that may express your intent better, or not:
- `{ [key: PropertyKey]: unknown }`
- `Record<string, unknown>`
- `Record<PropertyKey, unknown>` - this one's my preference.

Ditto for the other places where you use an index notation.